centralledger/chart-handler-timeout/configs/knexfile.js

```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
 module.exports = {
     client: 'mysql',
     connection: Config.DATABASE_URI,
+    min: Config.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
+    max: Config.DB_CONNECTION.DB_CONNECTION_POOL_MAX,
```

Comment:
This should be: `max: Config.DB_CONNECTION_POOL_MAX,`

Reply:
Updated as requested.